formats/cbor/commonTest/src/kotlinx/serialization/cbor/CborArrayTest.kt

```diff
@@ -0,0 +1,185 @@
+package kotlinx.serialization.cbor
+
+import kotlinx.serialization.*
+import kotlin.test.*
+
+
+class CborArrayTest {
+
+    @Test
+    fun writeReadVerifyArraySize1() {
+        /**
+         * 81    # array(1)
+         *    26 # negative(6)
+         */
+        val referenceHexString = "8126"
+        val reference = ClassAs1Array(alg = -7)
+
+        val cbor = Cbor(from = Cbor.CoseCompliant) {
+            useDefiniteLengthEncoding = true
```

Comment:
this flag is not needed here anymore as it is set in `CoseCompilant`. Same in several more places in this file

Reply:
fixed in 507ccd17619e42463e2fc146d7cf486f7793d03a